### PR TITLE
update finagle and twitter util versions

### PR DESCRIPTION
**Pull request by crispywalrus**
so we can start testing http2 for our finch based services. this
relies on finagle/finagle-oauth2#22 and assumes the version published
after that merge is 0.4.1

### CODE REVIEW

**Comment by vkostyukov** [comment]
I'm quite excited to hear you're trying out the h2 support! @crispywalrus 

Let's change finagle-oauth2 version to 0.6.43 so it's easier to identify which Finagle version it depends on.

File: build.sbt [hunk 1]
```diff
@@ -9,14 +8,14 @@ lazy val buildSettings = Seq(
...
 )
 
-lazy val finagleVersion = "6.42.0"
-lazy val utilVersion = "6.41.0"
-lazy val twitterServerVersion = "1.27.0"
-lazy val finagleOAuth2Version = "0.4.0"
+lazy val finagleVersion = "6.43.0"
+lazy val utilVersion = "6.42.0"
+lazy val twitterServerVersion = "1.28.0"
+lazy val finagleOAuth2Version = "0.6.43"
```
Comment (vkostyukov) [inline build.sbt]: finagle-oauth2 0.6.43 is on its way to MC.

Comment (crispywalrus) [inline build.sbt]: So it is in fact published, is there a way to make travis retry the build?

**Comment by vkostyukov** [comment]
It's green! Thanks @crispywalrus! Mind squashing the commits into one before I merge that?

**Comment by spockz** [comment]
@vkostyukov can't GitHub do that for you automatically?

**Comment by vkostyukov** [comment]
Idk, @spockz. I think it's one of those git operations that should be done manually in Github.

**Comment by crispywalrus** [comment]
Squash away!

**Pull request merged**